pom.xml

```diff
@@ -171,6 +171,7 @@
               <exclude>src/site/resources/download_lang.cgi</exclude>
               <exclude>src/test/resources/stringEscapeUtilsTestData.txt</exclude>
               <exclude>src/site/resources/release-notes/RELEASE-NOTES-*.txt</exclude>
+              <exclude>.travis.sh</exclude>
```

Comment:
You could put an Apache License comment at the top of this file because it accommodates comments.